app/decorators/backtrace_line_decorator.rb

```diff
@@ -64,7 +64,7 @@ def link_to_in_app_source_file(app, text)
   end
 
   def link_to_repo_source_file(app, text)
-    link_to_github(app, text) || link_to_bitbucket(app, text)
+    link_to_github(app, text) || link_to_bitbucket(app, text) || link_to_custom_backtrace_url(app, text)
```

Comment:
What do you think about putting the custom URL first? Seems like if you have one, you'd expect it to be used regardless.